tests/test_expval.py

```diff
@@ -132,7 +133,7 @@ def test_paulix_pauliy(self, theta, phi, varphi, qubit_device_3_wires, tol):
 
         expected = np.sin(theta) * np.sin(phi) * np.sin(varphi)
 
-        assert np.allclose(res, expected, tol)
+        assert np.allclose(res, expected)
```

Comment:
The function `test_paulix_pauliy` receives `tol` as a parameter. But I don't think it is in use anymore. I'm curious about why a change like this was not made in the `np.allclose` in functions before it?